ext/sockets/tests/socket_cmsg_credentials_fbsd.phpt

```diff
@@ -0,0 +1,92 @@
+--TEST--
+recvmsg(): receive SCM_CREDS messages
+--EXTENSIONS--
+sockets
+--SKIPIF--
+<?php
+
+if (!str_contains(PHP_OS, 'FreeBSD')) {
+    die('supported only on FreeBSD');
```

Comment:
You need to indicate whether the test should be skipped (`skip`) or marked as expected fail (`xfail`); in this case it should be skipped, so
```suggestion
    die('skip supported only on FreeBSD');
```